docarray/document/mixins/plot.py

```diff
@@ -92,7 +96,65 @@ def display(self, from_: Optional[str] = None):
         else:
             self.summary()
 
-    def display_tensor(self):
+    def _is_3d(self) -> bool:
+        """
+        Tells if Document stores a 3D object saved as point cloud or vertices and face.
+        :return: bool.
+        """
+        if self.uri and self.uri.endswith(tuple(MESH_FILE_EXTENSIONS)):
+            return True
+        elif self.tensor is not None and self.tensor.shape[1] == 3:
+            return True
+        elif self.chunks is not None:
+            name_tags = [c.tags['name'] for c in self.chunks]
+            if 'vertices' in name_tags and 'faces' in name_tags:
+                return True
+        else:
+            return False
+
+    def display_3d(self) -> None:
+        """Plot 3d data."""
+        from IPython.display import display
+        import trimesh
+
+        if self.tensor is not None:
+            # point cloud from tensor
+            from IPython import get_ipython
+
+            in_notebook = get_ipython().__class__.__name__ == 'ZMQInteractiveShell'
```

Comment:
also a CONSTANT here, I know is something we are not currently doing, but I would like to start changing this

Reply:
I am not a big fan of this check on the name of the class let me search for a better solution